Tests/Common/Securities/Options/OptionStrategiesTests.cs

```diff
@@ -1412,5 +1412,189 @@ public void BuildsBullPutLadderStrategy()
             Assert.AreEqual(expiration, lowStrikeLeg.Expiration);
             Assert.AreEqual(1, lowStrikeLeg.Quantity);
         }
+
+        [Test]
+        public void FailsBuildingCallBackspreadStrategy()
+        {
+            var canonicalOptionSymbol = Symbols.SPY_Option_Chain;
+            var underlying = Symbols.SPY;
+            var expiration = new DateTime(2023, 08, 18);
+
+            var strike1 = 300m;
+            var strike2 = 325m;
+
+            // Unordered and repeated strikes
+            Assert.Throws<ArgumentException>(
+                () => OptionStrategies.CallBackspread(canonicalOptionSymbol, strike2, strike1, expiration));
+            Assert.Throws<ArgumentException>(
+                () => OptionStrategies.CallBackspread(canonicalOptionSymbol, strike1, strike1, expiration));
+        }
```

Comment:
These could be test cases, like `[TestCase(300, 325)]` and `[TestCase(300, 300)]`.
Same for the rest of the tests